.drone.yml

```diff
@@ -2,4 +2,5 @@ pipeline:
     documentation:
         image: nextcloudci/documentation:documentation-4
         commands:
+            - make api-docs
```

Comment:
Best would be to include "api-docs" in "all", because we also use that on our docs system deployment. :wink: Let me have a look into this.

Reply:
Or even better: Include it in "make dev-html" ;) Then it's just at the right spot.